src/typeDefs/inputs.ts

```diff
@@ -282,6 +282,15 @@ export const inputs = gql`
     limit: PositiveInt!
   }
 
+  input UpdateAdvertisementInput {
+    id: ID!
```

Comment:
make this _id

Reply:
Please do the change as per the comment by @xoldyckk